skyportal/handlers/api/source.py

```diff
@@ -643,10 +650,30 @@ def get(self, obj_id):
               application/json:
                 schema: Error
         """
-        source = Source.get_obj_if_owned_by(obj_id, self.current_user)
+        source = Source.get_obj_if_owned_by(
```

Comment:
This is already an `Obj`, right? So no need to join on `Source.obj`

Reply:
Correct but this pattern is used throughout source.py, so keeping it here.